### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by maartenba**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,45 @@
...
-## Azure Toolkit for Eclipse
+* Sign in to an Azure account
+* Manage one or multiple subscriptions
+* Basic management of container hosts (create/delete, start/stop, view details)
+* List container registries
+* Basic management of Redis caches (create/delete, start/stop, view details, list keys/values)
+* Basic management of storage accounts (create/delete, list/create/delete blob container, list/upload/download/delete blobs)
+* Basic management of virtual machines (create/delete, start/stop, view details)
+* Basic management of web apps (create/delete, start/stop, view details)
+* Run configuration to deploy ASP.NET Core web app (can also provision SQL database)
```
Comment (sdubov) [inline README.md]: We do support Core web apps on any platform and full .Net framework web apps on Windows only.

File: README.md [hunk 2]
```diff
@@ -1,43 +1,45 @@
...
-* [Java Developer Center on Azure](https://docs.microsoft.com/en-us/java/azure/)
-* [Get Started for HDInsight](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-apache-spark-eclipse-tool-plugin)
+* [Issue tracker](https://github.com/JetBrains/azure-tools-for-intellij/issues)
+* [Plugin page](https://plugins.jetbrains.com/)
 
-## Data/Telemetry
-Azure Toolkits for Java collects usage data and sends it to Microsoft to help improve our products and services. Read our [privacy statement](https://go.microsoft.com/fwlink/?LinkID=620956) to learn more. If you don't wish to send usage data to Microsoft, you can turn off it in following places:
-* For **Eclipse**: Open `Window > Preferences > Azure`, and uncheck the checkbox.
-* For **IntelliJ IDEA**: Open `Settings/Preferences > Microsoft Tools > Azure`, and uncheck the checkbox.
+## History and differences with Microsoft Azure Toolkits for Java
```
Comment (sdubov) [inline README.md]: Their plugin is called "Azure Toolkit for IntelliJ"

File: README.md [hunk 3]
```diff
@@ -1,43 +1,45 @@
...
 
-## Contribution
-please see the [contribution instructions](CONTRIBUTING.md) if you wish to build the plugins from source.
+The Azure plugin for [JetBrains Rider](https://www.jetbrains.com/rider) is a fork of the [Microsoft Azure Toolkits for Java](https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-installation), available [on GitHub](https://github.com/Microsoft/azure-tools-for-java).
```
Comment (sdubov) [inline README.md]: Azure Toolkit for IntelliJ

File: README.md [hunk 4]
```diff
@@ -1,43 +1,45 @@
...
+
+Microsoft's Azure Toolkits for Java provide similar functionality to the Azure plugin for [JetBrains Rider](https://www.jetbrains.com/rider), however focus on the Java/JVM ecosystem and development flows. JetBrains [opened a pull request](https://github.com/Microsoft/azure-tools-for-java/pull/1725) to split base functionality (such as browsing Azure resources) from Java/JVM-specific features (such as deploying a `.war` file to the HDInsight service), which would open up the original plugin for various other ecosystems.
```
Comment (sdubov) [inline README.md]: Azure Toolkit for IntelliJ

File: README.md [hunk 5]
```diff
@@ -1,43 +1,45 @@
...
+* .NET-specific functionality, such as deploying an ASP.NET Core web application, has been added
+
+## Contributing
+
+Please see the [contribution instructions](CONTRIBUTING.md) if you wish to build the plugin from source.
 
 ## Disclaimer
-*azure-tools-for-java uses JxBrowser http://www.teamdev.com/jxbrowser, which is a proprietary software. The use of JxBrowser is governed by JxBrowser Product Licence Agreement http://www.teamdev.com/jxbrowser-licence-agreement. If you would like to use JxBrowser in your development, please contact TeamDev.*
+
+*azure-tools-for-java uses JxBrowser http://www.teamdev.com/jxbrowser, which is proprietary software. The use of JxBrowser is governed by JxBrowser Product Licence Agreement http://www.teamdev.com/jxbrowser-licence-agreement. If you would like to use JxBrowser in your development, please contact TeamDev.*
```
Comment (sdubov) [inline README.md]: Checking if we can drop that... build s running... :)

**Comment by IvanPashchenko** [comment]
We can rename the plugin to Azure tools for Rider (dropping IntelliJ)

**Comment by sdubov** [comment]
@IvanPashchenko, our plugin is called "Azure Toolkit for Rider". I've added comments about MS original plugin name.

File: README.md (hunk 5 above)
Comment (sdubov) [reply README.md]: @IvanPashchenko just merged that branch. We have dropped this dependency. :yay:

**Comment by maartenba** [comment]
Pushed some fixes, can you check once more?

File: README.md (hunk 5 above)
Comment (maartenba) [reply README.md]: Dropped

**Pull request merged**